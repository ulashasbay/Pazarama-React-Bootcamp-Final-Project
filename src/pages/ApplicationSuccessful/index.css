.app-success-container {
  font-family: "Open Sans", sans-serif;
  color: #3a3c47;
  line-height: 1.6;
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 0;
  padding: 0;
  height: auto;
}

.app-success-container h1 {
  margin-top: 20px;
  font-size: 30px;
}

.app-success-container b {
  font-weight: 700;
  margin-right: 6px;
}

.app-success-status {
  background: #f3fdf7;
  max-width: 400px;
  width: 100%;
  padding: 20px 44px 20px 44px;
  border: 1px solid #b0cac6;
  border-radius: 4px;
  box-shadow: 0 0 3px 0 rgba(41, 42, 46, 0.12);
  transition: all 0.3s ease;
  margin-bottom: 20px;
}

.success-message {
  background-color: #c4ffb8;
  font-size: 14px;
  border: none;
  border-radius: 3px;
  padding: 10px 0px 10px 16px;
  width: 98%;
}

@media (max-width: 960px) {
  .app-success-container {
    margin: 0 18px;
  }

  .app-success-status {
    border: none;
    box-shadow: none;
    padding: 20px 5px;
  }
}
