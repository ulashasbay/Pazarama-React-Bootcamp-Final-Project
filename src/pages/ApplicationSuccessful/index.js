import React from "react";
import AppInfoField from "../../components/AppInfoField";
import "./index.css";

function ApplicationSuccessful() {
  return (
    <div className="app-success-container">
      <span className="success-message">
        Başvurunuz başarıyla gerçekleşmiştir. Başvuru sürecinizi aşağıda size
        verilen başvuru numarası ile takip edebilirsiniz.
      </span>
      <h1>Başvuru Bilgileri</h1>
      <div className="app-success-status">
        <AppInfoField />
      </div>
    </div>
  );
}

export default ApplicationSuccessful;
