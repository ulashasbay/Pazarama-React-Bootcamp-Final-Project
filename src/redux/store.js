import { configureStore } from "@reduxjs/toolkit";
import appIdReducer from "./appId/appIdSlice";
import userInfoReducer from "./userInfo/userInfoSlice";
import isLoggedInReducer from "./isLoggedIn/isLoggedInSlice";

export const store = configureStore({
  reducer: {
    appId: appIdReducer,
    userInfo: userInfoReducer,
    isLoggedIn: isLoggedInReducer,
  },
});
