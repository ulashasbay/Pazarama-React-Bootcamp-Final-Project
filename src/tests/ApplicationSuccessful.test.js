import React from "react";
import ApplicationSuccessful from "../pages/ApplicationSuccessful";
import { BrowserRouter } from "react-router-dom";
import { Provider } from "react-redux";
import { store } from "../redux/store";
import renderer from "react-test-renderer";

test("ApplicationSuccessful renders correctly", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Provider store={store}>
          <ApplicationSuccessful />
        </Provider>
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
