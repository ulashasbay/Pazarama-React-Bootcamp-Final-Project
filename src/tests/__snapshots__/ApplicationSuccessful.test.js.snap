// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationSuccessful renders correctly 1`] = `
<div
  className="app-success-container"
>
  <span
    className="success-message"
  >
    Başvurunuz başarıyla gerçekleşmiştir. Başvuru sürecinizi aşağıda size verilen başvuru numarası ile takip edebilirsiniz.
  </span>
  <h1>
    Başvuru Bilgileri
  </h1>
  <div
    className="app-success-status"
  >
    <div
      className="row-app-show-page"
    >
      <b>
        Başvuru No:
      </b>
      
    </div>
    <div
      className="row-app-show-page"
    >
      <b>
        T.C. Kimlik No:
      </b>
    </div>
    <div
      className="row-app-show-page"
    >
      <b>
        Ad:
      </b>
    </div>
    <div
      className="row-app-show-page"
    >
      <b>
        Soyad:
      </b>
    </div>
    <div
      className="row-app-show-page"
    >
      <b>
        Yaş:
      </b>
    </div>
    <div
      className="row-app-show-page"
    >
      <b>
        Adres:
      </b>
    </div>
    <div
      className="row-app-show-page"
    >
      <b>
        Başvuru Nedeni:
      </b>
    </div>
    <div
      className="row-app-show-page"
    >
      <b>
        Başvuru Durumu:
      </b>
    </div>
    <div
      className="row-app-show-page"
    >
      <b>
        Başvuru Sonucu:
      </b>
    </div>
  </div>
</div>
`;
